os/net/mac/ble/ble-l2cap.h

```diff
@@ -73,7 +73,11 @@
 #ifdef BLE_L2CAP_CONF_NODE_MTU
 #define BLE_L2CAP_NODE_MTU      BLE_L2CAP_CONF_NODE_MTU
 #else
-#define BLE_L2CAP_NODE_MTU              1280
+#define BLE_L2CAP_NODE_MTU      PACKETBUF_SIZE
+#endif
+
+#if BLE_L2CAP_NODE_MTU != PACKETBUF_SIZE
+#error Invalid BLE node MTU setting
 #endif
```

Comment:
I know nothing about BLE MTU, but are you not allowed to have a smaller MTU than the packetbuf? If not, and `BLE_L2CAP_NODE_MTU` must indeed be equal to `PACKETBUF_SIZE`, would it not be cleaner to simply remove `BLE_L2CAP_CONF_NODE_MTU`, as it would be moot?

Reply:
Thanks for the comment. The PR has been updated and this restriction has been removed. Moreover, the default value is now PACKETBUF_SIZE instead of 1280.

Reply:
Don't you want to keep a check for for `BLE_L2CAP_NODE_MTU > PACKETBUF_SIZE` to provide a compile-time warning for misconfigurations? (if I understood the issue correctly)

Reply:
Yes, I added a check for this now.